app/admin-tab/host/route.js

```diff
@@ -5,7 +5,26 @@ export default Ember.Route.extend({
   endpoint: Ember.inject.service(),
 
   model: function() {
-    return this.get('store').find('setting', C.SETTING.API_HOST);
+    return this.get('store').findAll('setting').then((response) => {
+      var activeSettings = this.get('store').all('activesetting');
+      var out = Ember.Object.create({
```

Comment:
After `store.findAll('setting')` the settings service should be all populated, so you can just do 

``` return Ember.Object.create({
  host: settings.get(C.SETTING.API_HOST),
  catalog: settings.get(C.SETTING.CATALOG_URL)
})```

instead of getting activeSettings and looping over them again.